ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -56,6 +56,22 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.peer_groups[peer_group].next_hop_unchanged is arista.avd.defined(true) %}
    neighbor {{ peer_group }} next-hop-unchanged
 {%         endif %}
+{%         if router_bgp.peer_groups[peer_group].remove_private_as.enabled is arista.avd.defined(true) %}
```

Comment:
In my opinion, if `all` is true, we should still build the CLI command. Consider or condition if data model remains as is. It's still not obvious what should have more priority when `all: true` and `enabled: false`. Although check the doc template.

Reply:
This would not match other areas in AVD. it is fine to require `enabled: true` and just ignore the rest if not set correctly.

Reply:
ack. Let's focus on the data model anyway, as if we change it the problem may be fixed automatically